source/standards/logging.html.md.erb

```diff
@@ -1,108 +1,47 @@
 ---
-title: Logging
+title: Store and query logs
 expires: 2018-03-01
 ---
 
 # <%= current_page.data.title %>
 
 <%= partial :expires %>
 
-## Short-term querying of logs
+For incident management and debugging you should have a system in place so you can store and query your infrastructure and application logs.
 
-For the purposes of incident management and debugging, it's useful to
-have a system for storing and querying the most recent logs produced
-by your application and infrastructure.
+GDS has a shared account with [Logit](https://logit.io/), offering on-demand shared [ELK (Elasticsearch, Logstash and Kibana)](https://www.elastic.co/elk-stack) stacks as a service. ELK offers suitable short-term storage for your logs so they are both accessible and queryable.
 
-In general, tools for querying logs are not a good fit for long-term
-archival, and vice versa.
+You shouldn’t operate your own self hosted ELK stacks for logging. In the past we’ve found ELK relatively easy to set up, but long term it’s easier to use a cloud-hosted ELK solution.
 
-### Shared Elasticsearch, Logstash and Kibana (ELK) stack
+You should be aware that ELK isn’t appropriate for long term archival of logs. There are other services which are less expensive and more reliable.
 
-GDS has a shared account with [logit](https://logit.io/), which offers
-on-demand ELK stacks as a service.
+You can read the [Reliability Engineering team documentation](https://reliability-engineering.cloudapps.digital/#logging) for more information about using Logit.
 
-This is suitable for a short-term place to store logs so
-they are available and queryable.  This should not be considered a
-durable store of logs; it is *not* suitable for long-term archival of
-logs.
+## Store your logs
 
-Guides on how to use the logit service can be found in
-the [Reliability Engineering team documentation][].
+Your product may have legal or other requirements affecting how long you should store logs. For example, the [Payment Card Industry Data Security Standard (PCI-DSS}](https://en.wikipedia.org/wiki/Payment_Card_Industry_Data_Security_Standard) requires 3 months of easily accessible logs and 12 months of archived logs.
 
-### Sumo Logic
+### Short term storage
 
-Some teams have used [Sumo Logic][] successfully for storing and
-querying logs.
+You shouldn’t need to store logs spanning long periods in your short-term queryable store, practical retention periods for short-term queryable logs are:
 
-### Self-hosted Elasticsearch, Logstash and Kibana (ELK)
+* non-production environments - no more than 7 days
+* production environments - no more than 30 days
 
-Each team should not operate its own ELK stack for logging.
-In the past we've found that ELK is relatively easy to set up,
-but teams tend to not prioritise keeping it updated.
+### Long term archiving
 
-However, ELK isn't appropriate for long term archival of logs. There
-are other services which are less expensive and more reliable.
```

Comment:
we could do with an example or two of services which are less expensive and more reliable.

Reply:
@stephenharker @tomnatt can you suggest something here?